.eslintrc.js

```diff
@@ -34,6 +34,7 @@ module.exports = {
       files: ['*.ts', '*.tsx'],
       rules: {
         'react/prop-types': 'off',
+        'react/display-name': 'off',
```

Comment:
What does this do?

Reply:
Oh I meant to write a comment for that. Basically application.tsx is full of eslint errors because for the most part we are using arrow functions to define our components. Because of that we can't set a "display name" for debugging with dev tools. Since we aren't using that react feature consistently and it is hiding other errors from showing I turned it off

Reply:
OK sweet.